edge-modules/snitch/Cargo.toml

```diff
@@ -6,14 +6,20 @@ publish = false
 
 [dependencies]
 azure_sdk_for_rust = "0.9"
-chrono = "0.4"
+bytes = "0.4"
+byteorder = "1.2"
+chrono = { version = "0.4", features = ["serde"] }
 futures = "0.1"
+hex = "0.3"
 http = "0.1"
 hyper = "0.12"
+hyper-tls = "0.3.0"
+libflate = "0.1"
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
 serde_yaml = "0.7"
+tar = "0.4"
```

Comment:
Reminder to add to open source the modules we don't have. :) Sorry.

Reply:
Yeah, already have a task in the PBI for this.